src/ensembl/Dockerfile

```diff
@@ -1,13 +1,29 @@
-FROM node:14.16.0
+FROM node:14.17.0 AS builder
 
 ARG SOURCE_DIR="./"
 
 RUN mkdir -p /srv/ensembl-client
 
 COPY ${SOURCE_DIR} /srv/ensembl-client/
 
-# NOTE: NODE_ENV=development before npm install ensures that dev dependencies won't get skipped
-RUN cd /srv/ensembl-client/ && \
-    npm install --global npm@7.6.2 && \
+WORKDIR /srv/ensembl-client/
+
+RUN npm install --global npm@7.12.1 && \
     npm ci && \
-    npm run test
+    npm run test && \
+    npm run build
+
+
+# PRODUCTION IMAGE
+FROM node:14.17.0-alpine AS runner
+
+WORKDIR /srv/ensembl-client/
```

Comment:
Do we need to define this twice? (already defined on L9)

Reply:
Yep, we are inside a different container.